packages/catalog-model/examples/acme-corp.yaml

```diff
@@ -4,6 +4,5 @@ metadata:
   name: acme-corp
   description: A collection of all Backstage example Groups
 spec:
-  type: github
   targets:
-    - https://github.com/backstage/backstage/blob/master/packages/catalog-model/examples/acme/org.yaml
+    - ./acme/org.yaml
```

Comment:
Can't do this update and the other ones yet right? Would be breaking all existing usage in apps. I think giving the regular two weeks would be reasonable